pkgs/applications/editors/spacevim/init.nix

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I think we need some doc about this. Is there upstream documentation about the toml config file? Could you explain in a few short sentences how to map that to nix? Maybe we can link existing doc from some other project or copy parts of it.

Reply:
This PR has seen so many revisions :(
A long revision early ago just used the TOML configuration as defined by spacevim itself.

The TOML configuration is listed in their documentation https://spacevim.org/quick-start-guide/#configuration
The Nix expression should be a 1:1 mapping of it.

Reply:
```suggestion
# The Nix expression is a 1:1 mapping of it the spacevim toml config you can find here spacevim.org/quick-start-guide/#configuration

{
```

I think we should add something like this.